From 91dd130b705293551cef709b207faf8f90da35c8 Mon Sep 17 00:00:00 2001
From: DaoriJ92 <kuma4075@gmail.com>
Date: Wed, 22 May 2024 14:56:35 +0900
Subject: [PATCH] Initial commit

---
 README.md | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..21e88ab
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+# repo1
\ No newline at end of file
-- 
2.43.0.windows.1

